networktest.go

```diff
@@ -187,6 +187,12 @@ func (l *lightningNode) start(lndError chan error) error {
 		close(l.processExit)
 	}()
 
+	select {
+	case <-l.processExit:
+		return errors.New("lnd process was shutted down on the start")
+	case <-time.After(5 * time.Second):
```

Comment:
Hmm, `5 seconds` seems a bit much and can slow down the integration tests by a good bit since we generate several `lnd` instances and restart some several times. Would `1 second` suffice?